app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -151,58 +185,83 @@ docker pull kong:{{page.kong_versions[page.version-index].ce-version}}-alpine
  -p 8443:8443 \
  -p 127.0.0.1:8001:8001 \
  -p 127.0.0.1:8444:8444 \
- kong:latest</code></pre></div>
+ kong:latest
+ ```
 {% endnavtab %}
 {% endnavtabs %}
 {% endcapture %}
 {{ start_container | indent | replace: " </code>", "</code>" }}
 
-   {:.note}
-   > **Note**: The `HOSTNAME` for `KONG_PORTAL_GUI_HOST` should be preceded by a protocol, for example, `http://`.
-
-2. Verify your installation:
+    Where:
+    * `--name` and `--network`: The tag of the {{site.base_gateway}} image that
```

Comment:
`--name` is the name of the container to create. This can be anything.

The `kong/kong-gateway` entry at the end of the command is the image name and tag